waspc/data/Generator/templates/server/src/auth/utils.ts

```diff
@@ -154,7 +157,10 @@ export async function createUser(
           },
         }
       },
-    }
+    },
+    include: {
```

Comment:
We need the `Auth.id` to create a session

Reply:
I think I'd like to keep the `createUser` name since this is the purpose of this function. The return type encodes the info that they also receive the `auth` relation.